packages/ap-audit/alliance_platform/audit/settings.py

```diff
@@ -0,0 +1,71 @@
+from typing import TypedDict
+
+from alliance_platform.base_settings import AlliancePlatformSettingsBase
+from django.db.models import Value
+from django.db.models.expressions import Func
+from django.db.models.functions import Concat
+
+
+class AlliancePlatformAuditSettingsType(TypedDict, total=False):
+    """Settings for the Audit package of the Alliance Platform
+
+    These can be set in the Django settings file under the ``ALLIANCE_PLATFORM`` key:
+
+    .. code-block:: python
+
+        ALLIANCE_PLATFORM = {
+            "AUDIT": {
+                # settings go here
+            }
+        }
+    """
+
+    #: The name of the action used to evaluate permissions for auditing a model. Defaults to 'audit'.
+    LIST_PERM_ACTION: str | None
+    #: The name of the permission used to evaluate whether auditing of any type can be performed. If it resolves to ``False``,
+    #: this will override individual audit permissions determined by ``LIST_PERM_ACTION``. Defaults to
```

Comment:
I don't understand this comment about resolving to False. Type doesn't accept `False`, and it doesn't look like that would do anything anyway?